### DISCUSSION TITLE
Potential login problem

### DISCUSSION THREAD

**Discussion by maheshrijal** [ente-io/ente#136] [General]
Been using ente since the release, and loving it so far. That being said I've run into a problem. 

The sign in process involves the following:  Email OTP, 2FA OTP, and a password. 

Assuming that my 2FA is backed up somewhere safe & I remember the login password (No option of keeping the password in a password manager since it is required to login to the password manager itself). Ente still asks for an Email OTP before allowing me to fill password & 2FA.

The challenge for me is, the 2FA code for my Email is stored within ente, making it inaccessible until login. 

While I'm logged into ente & email on 2 devices at most times, assuming the worst case scenario I would be locked out of my account if I lose access to my both devices or email. 

Do I have any other options and what does your implementation look like?

**Comment by vishnukvmd** [comment]
Hey! We're working on making email verification optional during sign-in. I'll update this thread once the necessary changes are live.

**Reply by maheshrijal** [reply]
Ah that's great. Looking forward to it

**Reply by vishnukvmd** [reply]
Hey @maheshrjl, this has been fixed in our latest release. You can now opt-out of email-verification from the Settings section.
